app/src/main/java/com/zulip/android/LoginActivity.java

```diff
@@ -4,7 +4,6 @@
 import android.content.Intent;
 import android.content.IntentSender.SendIntentException;
 import android.os.Bundle;
-import android.support.annotation.Nullable;
```

Comment:
lgtm though if the unnecessary import was added in a previous commit in this branch probably should squash this into where it was introduced